R/error-checks.R

```diff
@@ -85,6 +85,14 @@ validateIsPositive <- function(object, nullAllowed = FALSE) {
   }
 }
 
+validateIsNonNegative <- function(object, nullAllowed = FALSE) {
+  validateIsOfType(object, c("numeric", "integer"), nullAllowed)

```

Comment:
I am going to add my commnents from previous here
We have a validateIsNumeric
And I would expect StrictPositive as opposed to NonNegative..but maybe that's just the optimistic guy in me